homeassistant/components/tibber/sensor.py

```diff
@@ -190,14 +181,14 @@ def __init__(self, tibber_home):
         """Initialize the sensor."""
         super().__init__(tibber_home)
         self._last_updated = None
-        self._is_available = False
-        self._extra_state_attributes = {}
+        self._attr_extra_state_attributes = {}
```

Comment:
This actually is a behavior we don't want in our entities.

This potentially causes the attribute to different on different moments in time, e.g., right now it is an empty dict, while later is contains keys.

The keys of the dict should be constant, any moment in time.